src/App.jsx

```diff
@@ -1,5 +1,13 @@
+import Homepage from "./pages/Homepage";
+import ListApp from "./pages/ListApp";
+
 export const App = () => {
-  return <div>Packing list</div>;
+  return (
+    <div>
+      <Homepage />
+      <ListApp />
+    </div>
+  );
```

Comment:
It would be more accurate to have ListApp added inside <HomePage> instead of underneath it.

Reply:
I added navigation because I want to expand functionality of ListApp in the future and also to practice using navigation in react. Can I leave it now as it is?

Reply:
I think it would be better to move it still. During lifecycle of the application, we want every iteration to be "deployable" which means, complete, and presentable to the user. Probably a better experience for you, if I also hold you to this requirement. So either have navigation implemented as porat of this PR, or cut `<ListApp />` from here and embed inside `<HomePage/>`.